src/user.ts

```diff
@@ -0,0 +1,232 @@
+type UserProperties = { [key: string]: unknown }
+
+export class User {
+    public userId: string
+    public orgIdToUserOrgInfo?: OrgIdToUserOrgInfo
+
+    // Metadata about the user
+    public email: string
+    public firstName?: string
+    public lastName?: string
+    public username?: string
```

Comment:
There are a few extra fields that are missing:
- pictureUrl
- hasPassword
- hasMfaEnabled
- createdAt